packages/core/phase1/triggers/TRPR0002.test.ts

```diff
@@ -0,0 +1,54 @@
+import TriggerCode from "bichard7-next-data-latest/dist/types/TriggerCode"
+import generateAhoFromOffenceList from "../../phase2/tests/fixtures/helpers/generateAhoFromOffenceList"
+import type { Offence } from "../../types/AnnotatedHearingOutcome"
+import TRPR0002 from "./TRPR0002"
+
+describe("TRPR0002", () => {
+  it.each([4575, 4576, 4577, 4585, 4586])(
```

Comment:
Nitpick: To avoid duplicating this array in our tests here, we can define a variable in this file and use that instead.

```
const resultCodes = [4575, 4576, 4577, 4585, 4586]

...

it.each(resultCodes)(...)

it.each(resultCodes)(...)
```

Reply:
👍